src/core/sequence_batch_scheduler.cc

```diff
@@ -1196,7 +1196,8 @@ OldestSequenceBatch::OldestSequenceBatch(
   Status status = DynamicBatchScheduler::Create(
       batcher_idx_, 1 /* runner_cnt */, GetCpuNiceLevel(config), OnInit,
       OnWarmup, OnSchedule, true /* dynamic_batching_enabled */,
-      false /* enforce_equal_shape_batch */, preferred_batch_sizes,
+      false /* enforce_equal_shape_batch */, false /* preserve_ordering */,
```

Comment:
The dynamic sequence scheduler always needs to preserve the order since it is sending in sequence requests. So change to "true".